### PR TITLE
Point users to the configuration page for overriding update locks

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: patch
See: https://jel.ly.fish/32cc2ee4-c5a7-4ec8-8e91-1ba4f1379bc4
See: https://www.flowdock.com/app/rulemotion/r-product/threads/9mk8QcdNp4UAwKeEUpXPHVxYM_c
See: https://github.com/balena-io/docs/pull/1929
Signed-off-by: Thodoris Greasidis thodoris@balena.io

### CODE REVIEW

**Comment by 20k-ultra** [comment]
@balena-ci rebase

File: docs/update-locking.md [hunk 1]
```diff
@@ -86,6 +86,9 @@ The update lock can be overridden in case you need to force an update, for insta
 
 To do this the device or fleet must have `BALENA_SUPERVISOR_OVERRIDE_LOCK` configuration variable set to "1".
 
-The easiest way to do this is to toggle the override in the actions section of your device. Go to the device you'd like to override a lock, then click the actions menu on the left hand side, and locate the 'Update lock override' toggle. Be sure to remove this toggle after if you'd like your update locks to work.
+The easiest way to do this is to use the 'Override the update lock' toggle in the [Fleet or Device Configuration][device-configuration] page. Go to the configuration page of the device you would like to override the lock, then locate the 'Override the update lock' item, click the activate button and then set the toggle to enabled. To get the update locks working again, disable this toggle afterwards.
```
Comment (klutchell) [inline docs/update-locking.md:89]:
```suggestion
The easiest way to do this is to use the 'Override the update lock ...' toggle in the [Fleet or Device Configuration][device-configuration] page. Go to the configuration page of the device or fleet, locate the 'Override the update lock ...' item, click the activate button, and set the toggle to enabled. Disable the toggle afterwards in order to restore the update locks.
```

Slight rewording to shorten the middle sentence with all the steps. Include ellipsis to indicate that there is more text in the UI following `Override the update lock ...`.

Comment (vipulgupta2048) [reply docs/update-locking.md:89]: I agree this reads better.

**Review by klutchell — APPROVED** [review]
Looks good!

**Pull request merged**